spec/services/discussion_service_spec.rb

```diff
@@ -1,26 +1,8 @@
 require 'rails_helper'
 
 describe 'DiscussionService' do
-  let(:ability) { double(:ability, :authorize! => true, can?: true) }
-  let(:user) { double(:user, ability: ability, update_attributes: true) }
```

Comment:
Double testing is like constructing your own little fantasy world in which you are the god of all unicorns and your code works. :goat: